cypress/features/regression/designSystem/simpleColorPicker.feature

```diff
@@ -17,4 +17,63 @@ Feature: Design System Simple Color Picker component
     Examples:
       | name                         | nameOfObject         |
       | mp150ú¿¡üßä                  | nameOtherLanguage    |
-      | !@#$%^*()_+-=~[];:.,?{}&"'<> | nameSpecialCharacter |
\ No newline at end of file
+      | !@#$%^*()_+-=~[];:.,?{}&"'<> | nameSpecialCharacter |
+
+  @positive
+  Scenario: When avaiableColors prop is provided changes rendnered colors
+    When I open "Design System Simple Color Picker Test" component page "default" in no iframe
+    Then It renders with all colors with "simpleColorPicker" json
+
+  @ignore
+  # there is no possibility to pass an array as an argument via url
+  Scenario: Color Picker renders all the provided colors and their respective labels
+    When I open default "Design System Simple Color Picker Test" component in noIFrame with "simpleColorSelect" json from "designSystem" using "availableColors" object name
+    Then It renders with all colors with "simpleColorPickerNew" json
+
+  @positive
+  Scenario: When on last color, going forward selects first color
```

Comment:
Could this scenario and the following one be put into an Examples table, or would it make it overly complex?

Reply:
done